man/cc-test-reporter-format-coverage.1.md

```diff
@@ -18,7 +18,7 @@ Locate, parse, and re-format supported coverage sources.
 
 ## -t, --input-type *COVERAGE TYPE*
 
-Tool used to generate the coverage data.
+(Optional) Identifies the input type (format) of the COVERAGE_FILE. If not passed, the input type is inferred based on file locations. Examples include the support sources listed below such as "simplecov" and "lcov".
```

Comment:
Please hard-wrap at 80 characters.

We don't say "Optional" or "Required" here. That's indicated above in

```
cc-test-reporter-format-coverage [--input-type=<coverage type>] [--output=<path>] [--prefix=<path>] [COVERAGE_FILE]
```

The `[brackets]` indication optional, requred arguments have no brackets.

For consistent language, I usually say "Defaults to ..." instead of "If not passed...", but that might be awkward to use here, so feel free to ignore this note.

I would remove the verbose *Examples include...* sentence, and instead just update the header to

```
## -t, --input-type *simplecov*|*lcov*|*...*
```

And show the complete list.